src/typings/react-native.d.ts

```diff
@@ -310,6 +317,11 @@ declare module 'react-native' {
         overScrollMode?: string; //enum( 'always', 'always-if-content-scrolls', 'never' )
         // iOS
         scrollIndicatorInsets?: {top: number, left: number, bottom: number, right: number };
+        // Windows only
+        onKeyDown? : Function;
+        onKeyUp? : Function;
+        tabNavigation? : string; // enum( 'local', 'cycle', 'once' );
```

Comment:
>string [](start = 25, length = 6)

I think you can use string types here:

tabNavigation?: 'local' | 'cycle' | 'once'

Reply:
ok

---
In reply to: [159357972](https://github.com/Microsoft/reactxp/pull/404#discussion_r159357972) [](ancestors = 159357972)